internal/tools/securitypolicy/helpers/helpers.go

```diff
@@ -145,6 +140,7 @@ func PolicyContainersFromConfigs(containerConfigs []securitypolicy.ContainerConf
 			workingDir,
 			containerConfig.ExpectedMounts,
 			containerConfig.Mounts,
+			containerConfig.AllowElevated,
```

Comment:
Would having a `ContainerPolicyFromConfig(*ContainerConfig) (*Container, error)` be easier than having to update all the callsites everytime new options are added?
Though it looks like you are passing the (expected) mounts manually rather than adding them to the config ....

Reply:
We made a decision to have the tooling call with explicit options, this way if we make a breaking change we get an error at compile time rather than at runtime.